pwndbg/wrappers/__init__.py

```diff
@@ -24,3 +28,6 @@ def _OnlyWithCommand(*a,**kw):
                 raise OSError('Could not find command %s in $PATH' % self.cmd_name)
```

Comment:
This currently does:
```
pwndbg> got 123
got: Could not find command checksec in $PATH (<class 'OSError'>)
```

Maybe we should change it to print for now @zachriggle ?